internal/service/odb/cloud_vm_cluster.go

```diff
@@ -476,6 +490,16 @@ func (r *resourceCloudVmCluster) Create(ctx context.Context, req resource.Create
 	plan.HostnamePrefixComputed = types.StringValue(*createdVmCluster.Hostname)
 	//scan listener port not returned by API directly
 	plan.ScanListenerPortTcp = types.Int32PointerValue(createdVmCluster.ListenerPort)
+	plan.GiVersionComputed = types.StringPointerValue(createdVmCluster.GiVersion)
+	giVersionMajor, err := getMajorGiVersion(createdVmCluster.GiVersion)
+	if err != nil {
+		resp.Diagnostics.AddError(
+			create.ProblemStandardMessage(names.ODB, create.ErrActionWaitingForCreation, ResNameCloudVmCluster, plan.DisplayName.ValueString(), err),
+			err.Error(),
+		)
+		return
+	}
+	plan.GiVersion = types.StringPointerValue(giVersionMajor)
```

Comment:
we should use the `flex` functions for value conversions instead of the framework `types

```suggestion
	plan.GiVersion = flex.StringtoFramework(cox, giVersionMajor)
```